recipes/custodian/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "custodian" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "30bb40a3eaa7dbbbfe01de74434d38ad4cdf4b5898b123b00631e9dcb194b5fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package defines a number of entry points that should be included here:
https://github.com/materialsproject/custodian/blob/master/setup.py#L44-L50

https://conda.io/docs/building/meta-yaml.html#python-entry-points